run.sh

```diff
@@ -1,8 +1,5 @@
-echo " >>>>>>>>  CLEANING UP EXISTING CONTAINERS "
```

Comment:
At first i thought these were merge conflicts. 😊

Reply:
noo , did that to put a unique message on the console whic the user can easily recognise